Extensions/xkit_patches.js

```diff
@@ -124,33 +124,74 @@ XKit.tools.parse_version = function(versionString) {
 	return version;
 };
 
+var blogs_get = document.createElement("script");
+blogs_get.textContent = "("+(function(){
+	var blogs = [];
+	var models = Tumblr.dashboardControls.allTumblelogs;
+
+	models.filter(function(model){
```

Comment:
If this is in Chrome, then I don't know the exact behavior. However, in Firefox, Tumblr is specifically aliased to unsafeWindow.Tumblr, which allows simple property accesses. See https://blog.mozilla.org/addons/2014/04/10/changes-to-unsafewindow-for-the-add-on-sdk/